typescript/nomad-deploy/src/bridge/checks.ts

```diff
@@ -81,3 +75,39 @@ export async function checkBridgeDeploy(
     ).to.equal(1, 'No eth helper found');
   }
 }
+
+/// Check bridge connections for an n-to-n setup (all connected to all)
+export async function checkBridgeConnections(deploys: AnyBridgeDeploy[]) {
+  for (const deploy of deploys) {
+    const bridgeRouter = deploy.contracts.bridgeRouter?.proxy!;
+    const remotes = deploys.filter(
+      (currDeploy) => deploy.chain.domain !== currDeploy.chain.domain,
+    );
+
+    remotes.forEach(async (remote) => {
+      const remoteDomain = remote.chain.domain;
+      const registeredRouter = await bridgeRouter.remotes(remoteDomain);
+      expect(registeredRouter).to.not.equal(emptyAddr);
+    });
+  }
+}
+
+/// Check bridge connections for an hub and spoke setup
+export async function checkHubAndSpokeBridgeConnections(
+  hub: AnyBridgeDeploy,
+  spokes: AnyBridgeDeploy[],
+) {
+  const hubRouter = hub.contracts.bridgeRouter?.proxy!;
+  spokes.forEach(async (spoke) => {
+    // Hub has registered spoke
+    const spokeDomain = spoke.chain.domain;
+    const hubRegisteredRouter = await hubRouter.remotes(spokeDomain);
+    expect(hubRegisteredRouter).to.not.equal(emptyAddr);
+
+    // Spoke has registered hub
+    const hubDomain = hub.chain.domain;
+    const spokeRouter = spoke.contracts.bridgeRouter?.proxy!;
+    const spokeRegisteredRouter = await spokeRouter.remotes(hubDomain);
+    expect(spokeRegisteredRouter).to.not.equal(emptyAddr);
+  });
+}
```

Comment:
made these separate functions and fixes what I think was bug with checks for hub/spoke vs n-to-n

Reply:
consider: checking that spokes aren't connected to each other

Reply:
https://github.com/nomad-xyz/nomad-monorepo/pull/58/commits/929ca746b8e70d437715a3ee57ad9739f3f39eb1